plugins/impcap/impcap.c

```diff
@@ -576,11 +576,11 @@ void packet_parse(uchar *arg, const struct pcap_pkthdr *pkthdr, const uchar *pac
 	struct tm timestruct;
 	time_t rawtime = time(NULL);
 	gmtime_r(&rawtime, &timestruct); // Get the current UTC date
-	strftime(strtime, 30, "%Y-%m-%dT%H:%M:%S%Z", timestruct); // Format the time as UTC str
```

Comment:
just a reminder: `strftime()` is not  safe to use as it depends on the "TZ" environment variable.

Reply:
Ok thank you.
What can I use safely instead ? 
I have to format the date as UTC to be compliant with MongoDB ISODate, that I use to send my impcap logs to.

Reply:
Unfortunately, there is no simple solution. IMHO linux timezone and time manipulation API is severely broken. I just wanted to make you aware of the fact. In our code, we do all the conversions ourselfs, but that limits us in regard to support for different TZ. :-(